plugins/link/dialogs/link.js

```diff
@@ -94,76 +94,53 @@
                 }, {
                     type: "vbox",
                     id: "urlOptions",
-                    children: [{
-                        type: "hbox",
-                        widths: ["25%", "75%"],
-                        children: [{
-                            id: "protocol",
```

Comment:
I talked with Kate Chipman about removing the "protocol" dialog and allowing the users to just paste in the URL into 1 box which is why this is removed